app/controllers/post_registration_controller.rb

```diff
@@ -1,5 +0,0 @@
-class PostRegistrationController < ApplicationController
-  before_action :authenticate_user!
```

Comment:
So :thinking: this was previously an authenticated path because the only way of getting the email for that page was from the state?

As we've now added an option for this coming in via an email param it doesn't need to be authenticated, right?

And if they arrive without any way of identifiying the email they're returned to root. Makes sense.